### ISSUE TITLE
Give the whole site a new design

### ISSUE THREAD

**Issue by yellowbrickc** [Virtual-Domain-driven-design/virtual-domain-driven-design#45]
The current site is functional but not appealing.

**Comment by aelgharniti** [comment]
@yellowbrickc @heimeshoff @Baasie: what do you think?
Prototype:
- v-1: https://www.figma.com/proto/1eDW0vIB4aO4ZLd5Idd0jV/VirtualDDD?node-id=172%3A49&scaling=min-zoom&page-id=172%3A46
- v-2: https://www.figma.com/proto/1eDW0vIB4aO4ZLd5Idd0jV/VirtualDDD?node-id=172%3A1291&scaling=min-zoom&page-id=172%3A47
- v-3: https://www.figma.com/proto/1eDW0vIB4aO4ZLd5Idd0jV/VirtualDDD?node-id=174%3A8&scaling=min-zoom&page-id=172%3A48

**Comment by Baasie** [comment]
@heimeshoff @yellowbrickc @Max-Git What are your thoughts.

I like v-3 very much for the main page!

**Comment by Baasie** [comment]
And thank you very much @aelgharniti and @Stefaniecespedes!! All look awesome

**Comment by Max-Git** [comment]
> @heimeshoff @yellowbrickc @Max-Git What are your thoughts.
> 
> I like v-3 very much for the main page!

Wow! What a surprise! V-3 is my favorite as well.

**Comment by yellowbrickc** [comment]
V-3 is my favorite too. What do you say @heimeshoff

**Comment by StefanieCespedes** [comment]
The design genius for the main page is @aelgharniti, but following his guidelines, I made a design prototype for the Learning DDD page. What do you think? @Baasie @heimeshoff @yellowbrickc @Max-Git

- [Version 1](https://www.figma.com/proto/1eDW0vIB4aO4ZLd5Idd0jV/VirtualDDD?node-id=231%3A0&scaling=min-zoom&page-id=227%3A0) - with a different hero image then the one used in the main page
- [Version 2](https://www.figma.com/proto/1eDW0vIB4aO4ZLd5Idd0jV/VirtualDDD?node-id=221%3A17&scaling=min-zoom&page-id=219%3A0) - with the same hero image used in the main page

Let me know what you think :)

**Comment by heimeshoff** [comment]
Thanks to the both of you, those are really great design prototypes.
For the landing page, I favour the contrast of v1, switching between the light backgrounds and the darker ones with blurred background images. I also like the tone of the backgroun-white. The glassmorphism is neat and also easy to implement, so again... big fan of number one. I do not like anything in particular from v2. V3 has my favourite hero design, though I would not give the Kandinsky image a shadow. I also like the sharp edges of all cards in v3 more, but I dislike the yellowish tint of the background white, gives it an old feel. In v3 the sponsors should not have glassmorphism but a white background, because transparend grey on yellowish white looks meh :)

For the learning DDD page I favour version 2 with the same comment, to get rid of the shadow of the Kandinsky image. Though I still dislike the yellowish tint of the background and would replace it with a whiter tone.

TL/DR, my choice would be:
- Take v1 as basis with the hero-image from v3
- Get rid of the shadow of the hero-image
- Use white and dark background succesively
- Use glassmorphism in areas with blurred images backgrounds
- Make cards with sharper edges

**Comment by yellowbrickc** [comment]
Thank you all! 

My problem is: I have no idea, I would like to extract myself from this decision. I am happy if you all come to an agreement. @StefanieCespedes @aelgharniti what do you think, could you create one design with the suggestions from @heimeshoff to have a final look?

Or is somebody else here desagreeing with those suggestions?

**Comment by Baasie** [comment]
@aelgharniti @StefanieCespedes I go with @heimeshoff on this one :). How should we proceed @heimeshoff? what are your thought, what do we need to start implementing your suggestion with the design?

**Comment by heimeshoff** [comment]
We need the images used in the background and hero, the rest we can eyeball of the figma pages and implement with tailwind in a breeze :)

**Comment by Baasie** [comment]
If we have that I smell an ensemble session coming up! Perhaps we can even make it into a meetup doing so =) @yellowbrickc @heimeshoff

**Comment by aelgharniti** [comment]
@heimeshoff @Baasie We can make the change, it's not a problem. We still need to create a design for mobile.

**Comment by Baasie** [comment]
@aelgharniti awesome thank you for the work!
